### PR TITLE
completed javascript-and-json

### PR DESCRIPTION
**Pull request by Blake-Frangoulis**
image

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+];
+
+console.log('library:', library);
+console.log('typeof library:', typeof library);
+
+const stringifyLibrary = JSON.stringify(library);
+console.log('stringifyLibrary:', stringifyLibrary);
+console.log('typeof stringifyLibrary:', typeof stringifyLibrary);
+
```
Comment (shawnkost) [inline javascript-and-json/main.js:25]: You're missing a whole step where you're supposed to create a JSON string manually and log the value & typeof

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing a step here Blake!

**Comment by Blake-Frangoulis** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**